templates/targets/x86_64/generic/flake.nix

```diff
@@ -43,6 +43,8 @@
     ghaf,
     nixpkgs,
     flake-utils,
+    # deadnix: skip
```

Comment:
outputs = inputs@ {bla, bla, ...} would probably be the more future proof method as it will include all attributes not explicitly dereferenced in this file and can be passed across more easily to other functions.

But as this is only a tmp fix we can go with the hardcoded.

Reply:
Done here: https://github.com/tiiuae/ghaf/pull/559